### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by aubinleyoung**
### In this project I:

- [ ] Added a column species of type string to my animal's table. 
- [ ] Insert the given data:
- [ ] Update the animals table by setting the species column to unspecified
- [ ] Inside a transaction:
  - Delete all animals born after Jan 1st, 2022.
  - Create a savepoint for the transaction.
  - Update all animals' weight to be their weight multiplied by -1.
  - Rollback to the savepoint
  - Update all animals' weights that are negative to be their weight multiplied by -1.
  - Commit transaction

- [ ] I took the following screenshot:
Screenshot 2023-03-28 174159
Screenshot 2023-03-28 203004
Screenshot 2023-03-28 204615
Screenshot 2023-03-28 204846
Screenshot 2023-03-28 213131
Screenshot 2023-03-28 213237
Screenshot 2023-03-28 213322
Screenshot 2023-03-28 213402
Screenshot 2023-03-28 213507
Screenshot 2023-03-28 213635

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -6,4 +6,12 @@ VALUES
...
-
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg, species) 
+VALUES 
+(5,'Charmander', '2020-02-08', 0, false, -11,'unspecified'),
+(6,'Plantmon', '2021-11-15', 2, true, -5.7,'unspecified'),
+(7,'Squirtle', '1993-04-02', 3, false, -12.13,'unspecified'),
+(8,'Angemon', '2005-06-12', 1, true, -45,'unspecified'),
+(9,'Boarmon', '2005-06-07', 7, true, 20.4,'unspecified'),
+(10,'Blossom', '1998-10-13', 3, true, 17,'unspecified'),
+(11,'Ditto', '2022-05-14', 4, true, 22,'unspecified');
```
Comment (Owusu-Desmond) [inline data.sql]: - [ ] From [the project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), the data that suppose to be inserted into the `animals` table has no values for the species column which means the species columns suppose to be null in the `animals` table :blush: but you set the value to `unspecified` :relaxed:

File: queries.sql [hunk 2]
```diff
@@ -7,3 +7,74 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5 ;
...
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+-- rollback to savepoint
+ROLLBACK TO my_savepoint;
+
+-- update all animals' weights that are negative to be their weight multiplied by -1
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+-- commit transaction
+COMMIT;
```
Comment (Owusu-Desmond) [inline queries.sql:57]: - [ ] Please make sure that anytime you complete a transaction you need to verify that the changes were made and add a screenshot with a label showing the result of each action. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @aubinleyoung , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @aubinleyoung 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**